src/views/Project/index.jsx

```diff
@@ -117,8 +108,20 @@ export default class Project extends Component {
             Bugs & Issues
           </Typography>
         </header>
-        <TasksTable items={data} />
+        {error && <ErrorPanel error={error} />}
+        {loading && <Spinner />}
+        {!error && !loading && <TasksTable items={data} />}
       </Fragment>
     );
   }
 }
+
+const ProjectClient = props => (
+  <Query
+    query={gql`{${githubQuery(props.match.params.project)}}\n${Issues}`}
```

Comment:
Are there extra curly braces inside this query?

Reply:
yes